Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:4.2
```

Comment:
The tools version should (almost) always be the minimum supported version. We could alternately maintain a separate manifest as described here: https://github.com/apple/swift-package-manager/blob/master/Documentation/Usage.md#version-specific-manifest-selection

But I was discouraged from doing so.